exercises/conversions.gie

```diff
@@ -101,7 +100,9 @@ Hints:
       https://proj4.org/operations/conversions/cart.html
 
     - Remember that `proj -le` returns a list of available ellipsoid models
-
+	
+	- Remember that the Hayford ellipsoid is known under a number of other
```

Comment:
You're making it too easy, man! :)

Reply:
:-) 

But having to know a century-old American geodesist by name, AND knowing by heart that "International, 1909" is the same as "International,1924" is probably forcing people to play geodetic scrabble...

I believe this is OK in the EUREF setting, since EUREF is a subcommission of the IAG. But considering that this material begins to take shape toward something that can be used in a more general setting, I figured it was a good idea to add an additional hint. For the benefit of cartographers, geographers, and other geodetically innocent people: One of them might be the future mother of your kids - better not scare her away up front :-)